lego/apps/files/views.py

```diff
@@ -46,6 +49,32 @@ def create(self, request, *args, **kwargs):
             status=status.HTTP_201_CREATED,
         )
 
+    @decorators.action(
+        detail=True,
+        methods=["PATCH"],
+        permission_classes=[permissions.IsAuthenticated],
+    )
+    def set_save_for_use(self, request, *args, **kwargs):
+        if request.user.has_perm(EDIT, Event) is False:
+            raise PermissionDenied()
+        file = self.get_object()
+        serializer = FileSaveForUseSerializer(data=request.data)
+        if serializer.is_valid():
+            if file.token != serializer.validated_data["token"]:
+                raise PermissionDenied()
+            try:
+                file.save_for_use = serializer.validated_data["save_for_use"]
+                file.save()
+            except KeyError:
+                return Response(
+                    {"This field is required!"}, status=status.HTTP_400_BAD_REQUEST
+                )
+            except Exception:
+                return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
Don't except `Exception` like this. Either catch the specific error or let it throw.